.eslintrc.cjs

```diff
@@ -24,6 +24,14 @@ module.exports = {
     extends: ['eslint:recommended'],
 
     overrides: [
+        // General
+        {
+            files: ['**/*.{ts,tsx}'],
+            rules: {
+                'no-empty': 'off',
+            },
+        },
+
```

Comment:
Note: this allows empty catch blocks without a pointless comment.

Before:
```ts
try { JSON.parse() } catch { /* duh */ }
```

After:
```ts
try { JSON.parse() } catch { }
```